composer.json

```diff
@@ -42,10 +42,10 @@
         "utopia-php/abuse": "0.7.*",
         "utopia-php/analytics": "0.2.*",
         "utopia-php/audit": "0.8.*",
-        "utopia-php/cache": "0.4.*",
+        "utopia-php/cache": "0.5.*",
         "utopia-php/cli": "0.12.*",
         "utopia-php/config": "0.2.*",
-        "utopia-php/database": "0.15.*",
+        "utopia-php/database": "dev-feat-cache-upgrate as 0.15.1",
```

Comment:
Reminder to use the correct library version 👍

Reply:
Updated